ReactNativeClient/lib/models/Setting.js

```diff
@@ -172,7 +172,9 @@ class Setting extends BaseModel {
 			},
 
 			'sync.3.auth': { value: '', type: Setting.TYPE_STRING, public: false },
+			'sync.3.accountProperties': { value: '', type: Setting.TYPE_STRING, public: false },
 			'sync.4.auth': { value: '', type: Setting.TYPE_STRING, public: false },
+			'sync.4.accountProperties': { value: '', type: Setting.TYPE_STRING, public: false },
 			'sync.7.auth': { value: '', type: Setting.TYPE_STRING, public: false },
```

Comment:
The account properties should normally go in "sync.3.context", for consistency. Please could you move it there?